sherpa/data.py

```diff
@@ -1986,6 +2019,63 @@ def xhi(self):
 
 
 class Data2D(Data):
+    '''2D data set
+
+    This class represents a 2D data set. It is desigend to work with flattened
+    arrays for coordinates and independent variables, which makes it easy to
+    deal with filters and sparse or irregularly-placed grids.
+
+    Of course, the same structure can also be used for regularly-gridded data,
+    it just has to be passed in as a flattened array. However, in this case
+    the more specialiazed `DataIMG` class might be more appropriate.
+
+    Parameters
+    ----------
+    name : string
+        name of this dataset
+    x0 : array-like
+        Coordinates in one dimension
```

Comment:
Perhaps

```suggestion
        Coordinates of the first dimension of the independent axis.
```

but this is also ungainly and doesn't quite sound right